services/indexes/models/avax.go

```diff
@@ -70,17 +70,16 @@ type Asset struct {
 	ID      StringID `json:"id"`
 	ChainID StringID `json:"chainID"`
 
-	Name         string `json:"name"`
-	Symbol       string `json:"symbol"`
-	Alias        string `json:"alias"`
-	Denomination uint8  `json:"denomination"`
```

Comment:
The go deserializer is a bit of magic.
Does re-ordering these cause issues?

Reply:
It would cause compile time errors for and only for keyless instantiations of this type but we don't have any.